front-end/peer-prep/src/index.tsx

```diff
@@ -61,6 +62,7 @@ root.render(
                     </Route>
                     {/* All non-protected routes are written here */}
                     <Route path="SignIn" element={<SignInPage/>} />
+                    <Route path="/profile" element={<UserPage/>}/>
```

Comment:
Please move this route under the `protected routes` under line 59 as only authenticated users should have access to the `UserPage`!

Reply:
Alright done, thanks for the feedback! Also, I changed the path to /user to keep the naming consistent.

Reply:
Thank you too for keeping the naming consistent!